etl/tg-jdbc-driver/tg-jdbc-restpp/src/main/java/com/tigergraph/jdbc/driver/QueryParser.java

```diff
@@ -762,14 +772,14 @@ public HttpRequestBase buildQuery(String host, Integer port, Boolean secure,
       case HttpPost:
         HttpPost post = new HttpPost(url);
         if (!"".equals(json)) {
-          if (!"".equals(this.job)) {
-            StringEntity payload = new StringEntity(json, ContentType.TEXT_PLAIN);
-            post.setEntity(payload);
-          } else {
-            StringEntity payload = new StringEntity(json, ContentType.APPLICATION_JSON);
-            post.setEntity(payload);
+          StringEntity payload = new StringEntity(json, "UTF-8");
+          if ("".equals(this.job)) {
+            payload.setContentType("application/json");
           }
+          payload.setContentEncoding("UTF-8");
```

Comment:
L779(780) and L782/3: I saw the first is payload and 2nd is "this.payload". Are they the same?

Reply:
They are not the same. "this.payload" is a class member variable, "payload" is a local variable. Will rename the former one in next release.